index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div>
+      <nav>
+        <h3>My logo</h3>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </div>
+    <div class="landing">
+      <h1>Hey There.</h1>
+      <h1>I'm Alex.</h1>
+      <h3>i'm a software developer</h3>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
+      <div class="icons">
+        <i class="fa-brands fa-github"></i>
+        <i class="fa-brands fa-linkedin-in"></i>
+        <i class="fa-brands fa-angellist"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fa-solid fa-m"></i>
+      </div>
```

Comment:
- [ ] It is better to use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
- [ ] Also, please customize this section to have links to your social media account as this is your portfolio and not just a template